std_msgs/Header header

float64[3] left_coefficients
float64[3] right_coefficients
float64[9] transformation_matrix